docs/collections/_other/doc-history.md

```diff
@@ -4,18 +4,28 @@ title: Document history
 nav_order: 14
 ---
 
-# Document history for the Cedar Policy Language Guide {#doc-history}
+# History for the Cedar Policy Language {#doc-history}
 {: .no_toc }
 
+## Document history for the Cedar Policy Language Guide 
 The following table describes major documentation updates for Cedar.
 
-| Cedar<br/>Version | Description | Date |
-| --- |--- |--- |
-| - | Added [Entities & context syntax](../auth/entities-syntax.html) topic | July 27, 2023 |
-| - | Added [Schema grammar](../schema/schema-grammar.html) topic | July 17, 2023 |
-| - | Added [Best Practices](../overview/best-practices.html) topics | July 14, 2023 |
-| - | Added [JSON policy format](../policies/json-format.html) reference page | July 14, 2023 |
-| 2.3 | Implements [RFC 9](https://github.com/cedar-policy/rfcs/pull/9) that disallows whitespace in identifiers | June 29, 2023 |
-| 2.2 | Changes to Cedar API that don't affect the policy language | May 30, 2023 |
-| 2.1 | Changes to Cedar API that don't affect the policy language | May 25, 2023 |
-| 2.0 | Initial release of the Cedar Policy Language Guide | May 10, 2023 | 
+| Description | Date |
+| --- | --- |
+| Added [Schema grammar](../schema/schema-grammar.html) topic | - | July 17, 2023 |
+| Added [Entities & context syntax](../auth/entities-syntax.html) topic | - | July 27, 2023 |
+| Added [JSON policy format](../policies/json-format.html) reference page | - | July 14, 2023 |
+| Added [Best Practices](../overview/best-practices.html) topics | - | July 14, 2023 |
+
+
+## History of the Cedar language
+
+| Cedar<br/>Version | Description | Cedar SDK<br/>Versions | Date | 
+| --- |--- |--- | --- |
+| 2.1.2 | Change to how validation treats template-linked policies | 2.4.2 | October 23, 2023 |
+| 2.1.1 | Increased validation precision | 2.4.0 - 2.4.1 | September 29, 2023 |
+| 2.1.0 | RFC #9: Disallows whitespace in namespaces | 2.3.0 - 2.3.3 | June 29, 2023 |
```

Comment:
Add a link to the RFC (and possibly the PRs for the other breaking changes, if you think that's appropriate):
* RFC9: https://github.com/cedar-policy/rfcs/blob/main/text/0009-disallow-whitespace-in-entityuid.md
* Increased precision when validating records: https://github.com/cedar-policy/cedar/pull/117
* Change to how validation treats template-linked policies: https://github.com/cedar-policy/cedar/pull/371